docs/release-notes/release-notes-0.9.7.md

```diff
@@ -0,0 +1,162 @@
+Also "restarting" is now the same as starting; it will operate based on whatever mixdepths are funded, there is no "starting mixdepth".
+For details see the substantially updated [tumbler guide](../tumblerguide.md) which now has algorithm examples, and the explanation of the fix and change in the [PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1324).
+
+* `d3dc9d7` Change tumbler algo to cycle and add tests, and `d0bf888` `7ffc747` `e5ed7f2` `c0df868`
+* `b4e4f2a` susbstantial rewrite of tumblerguide.md
+* `524cbda` update Qt for new tumbler algo
+* `75c444e` Update wallet_rpc for new tumbler code
+
+### Disallowing utxos with unconfirmed spends
+
+This fix is a bit technical but very important. The TLDR is that it should remove a common issue of makers sending spent utxos to takers as candidates for coinjoins, which affected some users. See [the PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1295) for some detailed discussion.
+
+`a3e1ba3` Default 3rd argument of gettxout should be True
+
+### New script to calculate fidelity bond values
+
+This new script is intended to help users make more informed choices about fidelity bonds. It gives data on the value of the bond dependent on locktime and amount, and optionally can compare with existing bonds in the orderbook (using a json export of the latter).
+
+* `72bf447` Script: Add bond-calculator.py
+
+### New display of 'available' balance
+
+This new feature in the UI (both CLI and Qt) displays tags 'FROZEN', 'PENDING', 'LOCKED' to indicate the status of different utxos (with 'PENDING' meaning unconfirmed). Available balance is distinguished from total balance with parentheses.
+As you can see from this description already, the situation is substantially more complex for Joinmarket than for simpler wallets because there are at least 3 ways in which utxos' usability might be restricted, and this is not necessarily the same for all use cases (e.g. maker vs taker). Still, this UI addition gives the user a lot more information.
+This additional 'available balance' information is now also returned in the RPC API.
+
+* `015cb4a` If there is unavailable fund, display 2 balances: total balance and unlocked balance.
+* `ea6c34d` update yaml for new available_balance fields
+* `75a93df` fix available_balance
+
+We also had to update the json-ified wallet display to accommodate this:
+
+* `08581f8` Fixes json serialization of wallet display
+
+### RPC-API updates
+
+There continue to be several small updates to the API, most notable this time is the addition of schedule status to the `/session` endpoint.
+
+* `8ee6b39` Report offerlist in /session call if possible
+* `b2ab4db` add nickname field to session response
+* `2cb41fc` review: reaqd schedule from taker object, also `190d56e` `57d1476`
```

Comment:
```suggestion
* `2cb41fc` review: read schedule from taker object, also `190d56e` `57d1476`
```

Reply:
These two typos were in the commit comment itself. I'll just leave it.